### PR TITLE
Removed deprecated API method and code warning, updated to Swift 5

### PR DESCRIPTION
**Pull request by domeniconicoli**
-Removed deprecated API methods
-Updated to Swift 5 version
-Removed many code warnings

### CODE REVIEW

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -251,15 +251,17 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0700;
-				LastUpgradeCheck = 0730;
+				LastUpgradeCheck = 1100;
 				ORGANIZATIONNAME = "Zhixuan Lai";
 				TargetAttributes = {
 					09673A7D1AEF5B2E008C7240 = {
 						CreatedOnToolsVersion = 6.3.1;
+						DevelopmentTeam = 89D52653SY;
```
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: could you remove this change?

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -251,15 +251,17 @@
...
 						LastSwiftMigration = 0900;
 					};
 					09673A921AEF5B2E008C7240 = {
 						CreatedOnToolsVersion = 6.3.1;
+						DevelopmentTeam = 89D52653SY;
```
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: this one too

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -538,37 +564,41 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 4BA34855EB4A4C2074C86E94 /* Pods-ZLSwipeableViewSwiftDemo.debug.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 89D52653SY;
```
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: ditto, delete please

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -538,37 +564,41 @@
...
 			name = Debug;
 		};
 		09673A9F1AEF5B2E008C7240 /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 4AEF23DC622C3A7E809F3742 /* Pods-ZLSwipeableViewSwiftDemo.release.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 89D52653SY;
```
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: all of the ones that add your development team, there are a couple more in the file

File: ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -591,7 +621,9 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = C998BF9C49543ECF772E8671 /* Pods-ZLSwipeableViewSwiftDemoTests.release.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```
Comment (AndrewSB) [inline ZLSwipeableViewSwiftDemo/ZLSwipeableViewSwiftDemo.xcodeproj/project.pbxproj]: is this required?

**Review by AndrewSB — CHANGES_REQUESTED** [review]
this is super helpful, thanks for taking the time. we can bump the version after this PR is merged. take a look at my notes and I’m happy to merge after you’ve made the changes I requested

**Comment by domeniconicoli** [comment]
Hi Andrew,
I made the requested changes.
Let me know if everything is fine now.

Best Regards.

**Comment by AndrewSB** [comment]
pushed a new release: https://github.com/zhxnlai/ZLSwipeableViewSwift/releases/tag/0.0.9

**Pull request merged**